### PR TITLE
add quantitative_indices

### PR DESCRIPTION
**Pull request by JaafarHammoud**

### CODE REVIEW

File: eXNN/InnerNeuralTopology/quantitative_indices.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Compute the persistent entropy and normed persistent entropy
+def compute_entropy(lengths, length_sum):
+    return (-1) * np.sum(np.divide(lengths, length_sum)*np.log(np.divide(lengths, length_sum)))
+
+def compute_normed_entropy(entropy, length_sum):
+    return entropy / np.log(length_sum)
+    
+# Compute births
+def compute_births(diag):
+    return np.array([x[1] for x in diag[0]])
```
Comment (IvanTomilov1) [inline eXNN/InnerNeuralTopology/quantitative_indices.py:39]: There and in the next function is needed double-check - birth is start of bar in barcode, and death is it's end.

Comment (tanyapole) [reply eXNN/InnerNeuralTopology/quantitative_indices.py:39]: infra comment

Comment (IvanTomilov1) [reply eXNN/InnerNeuralTopology/quantitative_indices.py:39]: For each dimension there are different barcodes, so it's true that you need to calculate metrics first each dimension separately.
About first part - okay, it's alright, there is no error

Comment (JaafarHammoud) [reply eXNN/InnerNeuralTopology/quantitative_indices.py]: @IvanTomilov1 
I keeped the two functions `compute_births` and `compute_deaths` and added two function `compute_birth` and `compute_death` which take two parameter (the diagram and the dimension that we want to calulte the birth and death for it)

**Review by tanyapole — APPROVED** [review]

**Review by MedAI-Lab-ITMO — APPROVED** [review]

**Pull request merged**